project/src/main/webapp/document.jsp

```diff
@@ -19,8 +19,9 @@
     <script src="https://firepad.io/releases/v1.5.9/firepad.min.js"></script>
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@0.9.0/css/bulma.min.css" />
     <link rel="stylesheet" href="main.css" />
-    <script type="module" src="./components/toolbar-component.js"></script>
-    <script type="module" src="./components/share-component.js"></script>
+    <script type="module" src="./components/document/toolbar-component.js"></script>
```

Comment:
for sure going to run into some merge conflicts when we add commenting, you'll have to help me sort them out and get all my js functions into another file

Reply:
we can just run *rebase master* to work this out; I'll keep this folder structure since it modularizes our components